go/test/endtoend/onlineddl/scheduler/onlineddl_scheduler_test.go

```diff
@@ -1806,26 +1806,39 @@ func testScheduler(t *testing.T) {
 		sqls := []string{
 			`alter table t1_test force`,
 			`alter table t2_test force`,
+			`drop table if exists non_existent_1`,
+			`drop table if exists non_existent_2`,
 		}
 		sql := strings.Join(sqls, ";")
 		var vuuids []string
 		t.Run("apply schema", func(t *testing.T) {
 			uuidList := testOnlineDDLStatement(t, createParams(sql, ddlStrategy+" --in-order-completion --postpone-completion --allow-concurrent", "vtctl", "", "", true)) // skip wait
 			vuuids = strings.Split(uuidList, "\n")
-			assert.Len(t, vuuids, 2)
+			assert.Len(t, vuuids, 4)
 			for _, uuid := range vuuids {
 				waitForReadyToComplete(t, uuid, true)
 			}
+			for i, uuid := range vuuids {
+				rs := onlineddl.ReadMigrations(t, &vtParams, uuid)
+				require.NotNil(t, rs)
+				for _, row := range rs.Named().Rows {
+					inOrderCompletionPendingCount := row.AsUint64("in_order_completion_pending_count", 0)
+					assert.EqualValues(t, i, inOrderCompletionPendingCount)
```

Comment:
You should be using the same type. `i` is an int while `inOrderCompletionPendingCount` is a uint64.

Reply:
@mattlord thanks Matt, that should address at least 1 x of the problems with tests 👍. Added